src/lib/viewers/controls/settings/SettingsList.tsx

```diff
@@ -3,7 +3,11 @@ import classNames from 'classnames';
 import noop from 'lodash/noop';
 import { decodeKeydown } from '../../../util';
 
-export type Props = React.HTMLAttributes<HTMLDivElement> & { isActive?: boolean };
+export type Props = React.HTMLAttributes<HTMLDivElement> & {
+    className?: string;
```

Comment:
It looks like the `React.HTMLAttributes` type no longer contain these common React attributes? Can we use a different type, instead, to avoid having to declare them explicitly?

Reply:
Hm, on closer inspection it looks like classname should be on this type - https://github.com/DefinitelyTyped/DefinitelyTyped/blob/4e99c4c87d11cf464312b6150a1e5001be4f809b/types/react/index.d.ts#L1923C1-L1924C1. Seems like other people also ran into this issue and found a workaround: https://github.com/jsx-eslint/eslint-plugin-react/issues/3284. I'll give this a try..